build/Targets/Settings.props

```diff
@@ -124,6 +124,7 @@
     <When Condition="'$(MSBuildProjectExtension)' == '.vbproj' OR '$(Language)' == 'VB' OR '$(ProjectLanguage)' == 'VB'">
       <PropertyGroup>
         <ProjectLanguage>VB</ProjectLanguage>
+        <LangVersion>Latest</LangVersion>
```

Comment:
Also, technically this is the VB section, and should probably say "15.3", and the one on line 166 should say 7.1 😄